io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -251,6 +251,26 @@ impl From<LvsError> for tonic::Status {
             LvsError::WipeFailed { source } => source.into(),
             LvsError::ResourceLockFailed { .. } => Status::aborted(e.to_string()),
             LvsError::MaxExpansionParse { .. } => Status::invalid_argument(e.to_string()),
+            LvsError::BdevRescanFailed { .. } => {
+                let mut status = Status::failed_precondition(e.to_string());
+                status.metadata_mut().insert(
+                    "bdev_rescan_failed",
+                    tonic::metadata::MetadataValue::from(0),
+                );
+                status
+            }
+            LvsError::BdevNotExtended { .. } => {
+                let mut status = Status::failed_precondition(e.to_string());
```

Comment:
This is not `failed_precondition` right? This should be failed/aborted/terminated/nternal right?